decidim-admin/spec/shared/manage_process_steps_examples.rb

```diff
@@ -97,7 +98,7 @@
     let!(:process_step2) { create(:participatory_process_step, participatory_process: participatory_process) }
 
     before do
-      visit decidim_admin.participatory_process_path(participatory_process)
+      visit current_path
```

Comment:
The one set at the `before` part on top of the file. I need to do this reload in order to show the newly created record.